migrations/create_db.sql

```diff
@@ -1,5 +1,10 @@
 DROP DATABASE IF EXISTS spark;
 
+# Implicitly create the user and then drop the user.
+GRANT USAGE ON *.* TO 'spark'@'localhost' IDENTIFIED BY 'password';
+DROP USER 'spark'@'localhost';
+flush privileges;
```

Comment:
This 3 lines are the same as "drop user if exists", which does not exists in mysql (up until last version, and at this point of time I can't rely on it)